DEVELOPER_GUIDE.md

```diff
@@ -986,3 +989,34 @@ POST /api/opensearch-dashboards/index_patterns
   ]
 }
 ```
+
+## Submit pull request
+### Before submit a pull request
+First-time contributors should head to the [contributing guide](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/CONTRIBUTING.md) to get started.
```

Comment:
Even for experience developer is it worth to read it again. Maybe we could add to PR check list.